Common/include/option_structure.inl

```diff
@@ -1277,6 +1277,78 @@ public:
   }
 };
 
+class COptionInletScalar : public COptionBase {
+  string name; // identifier for the option
+  unsigned short & size;
+  string * & marker;
+  su2double ** & inletscalarval; 
```

Comment:
if this is a completely new type of option please use vector<vector< instead of ** and if the number of columns is constant use an su2activematrix